apps/sensenet/cypress/integration/ui/addNewMenu.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+context('AddNew Menu', () => {
+  before(() => cy.clearCookies({ domain: null } as any))
+
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('should open a dropdown with the list of allowed child types', () => {
+    const dropdownItems = [
+      'Folder',
+      'Document Library',
+      'Image Library',
+      'Event List',
+      'Memo List',
+      'Link List',
+      'Task List',
+      'Custom List',
+      'Workspace',
+      'System Folder',
+      'Demo Workspace',
```

Comment:
```suggestion
      'Demo Workspace',
      'Image'
```
It is because the list on the admin surface is changed after the steps in the issue was made, so